hack/installers/install-gotestsum.sh

```diff
@@ -21,6 +21,6 @@ url=https://github.com/gotestyourself/gotestsum/releases/download/v${gotestsum_v
 
 mkdir -p /tmp/gotestsum-${gotestsum_version}
 tar -xvzf ${temp_path} -C /tmp/gotestsum-${gotestsum_version}
-cp /tmp/gotestsum-${gotestsum_version}/gotestsum ${DIST_PATH}/gotestsum
-chmod +x ${DIST_PATH}/gotestsum
+sudo cp /tmp/gotestsum-${gotestsum_version}/gotestsum ${INSTALL_PATH}/gotestsum
+chmod +x ${INSTALL_PATH}/gotestsum
```

Comment:
This may require a sudo because `$BIN` is set to `/usr/local/bin`

Reply:
Thanks for the catch! Added sudo before changing permissions as well.